sites/statics/package.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Is this intended to be published to npm? If not, you may want to make it a private package.

Reply:
Oops, forgot that step. Definitely.